packages/wonder-blocks-dropdown/components/separator-item.js

```diff
@@ -22,25 +22,23 @@ export default class SeparatorItem extends React.Component<{|
      */
     style?: StyleType,
 |}> {
-    static __IS_SEPARATOR_ITEM__ = true;
     static isClassOf(instance: React.Element<any>) {
         return instance && instance.type && instance.type.__IS_SEPARATOR_ITEM__;
     }
+
     static __IS_SEPARATOR_ITEM__ = true;
 
     render() {
         return (
-            //  wrapper to use styles from react-window (if applies)
-            <View style={this.props.style}>
-                <View style={styles.separator} />
-            </View>
+            // pass optional styles from react-window (if applies)
+            <View style={[styles.separator, this.props.style]} />
         );
     }
 }
 
 const styles = StyleSheet.create({
     separator: {
-        backgroundColor: Color.offBlack16,
+        boxShadow: `0 -1px ${Color.offBlack16}`,
```

Comment:
Oh, nice! What's the effect of this?

Reply:
Before that we used `height: 1px` to fill the space with the `background` color,  but `react-window` was overriding that value to `9px`, b/c of the SEPARATOR_ITEM_HEIGHT that is 1px (border) + 8px (vertical margin).

This change prevents that behavior.